src/emulator/CMakeLists.txt

```diff
@@ -1,10 +1,22 @@
 set(VITA3K_ORG_NAME "Vita3K")
 set(VITA3K_APP_NAME "Vita3K")
+set(VITA3K_APP_VERSION "v0.1")
```

Comment:
We should build this from the current git tag. That way, the tag is the source of truth and we don't have to manually keep this file updated with the tag string.

Reply:
Hmm yes i think is good idea, but caused problem also for built without git, and i don't really know who set this with git tag hmm